applications/crossbar/src/cb_mobile_manager.erl

```diff
@@ -0,0 +1,94 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2010-2015, 2600Hz
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(cb_mobile_manager).
+
+-export([delete_account/1]).
+
+-include("./crossbar.hrl").
+
+-define(MOD_CONFIG_CAT, <<"mobile_manager">>).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%
+%% @end
+%%--------------------------------------------------------------------
+delete_account(Context) ->
+    AccountId = cb_context:account_id(Context),
+    AuthToken = cb_context:auth_token(Context),
+
+    UrlString = req_uri([<<"accounts">>, AccountId]),
+    Headers = req_headers(AuthToken),
+
+    lager:debug("mobile_manager delete via ~s", [UrlString]),
+
+    Resp = ibrowse:send_req(UrlString, Headers, 'delete'),
+    handle_resp(Resp).
```

Comment:
I really like this API. Could you create a whistle "rest" module that exports your private functions
* `req_uri/1`
* `req_headers/1`
* `send_req/3`
* `handle_resp/1`
* `decode/1`
?

This way we'd have an adapter and we could switch to whatever httpc library we'd like (hackney, httpc, whatevs).
Also, I know a lot of places where this same code is duplicated (or should be used anyway), so I vote for a new REST utils module.